packages/ruby/lib/content_type_helper.rb

```diff
@@ -10,6 +10,9 @@ module ContentTypeHelper
   ]
 
   def json?
-    JSON_MIME_TYPES.include? content_type
+    JSON_MIME_TYPES.any? do |mime_type|
+      escaped_mime_type = Regexp.escape(mime_type)
+      content_type.match?(escaped_mime_type)
```

Comment:
Since we are matching on a static string here, would it be cleaner to rely on `String#include?` instead of dealing with Regex?

Reply:
Oh yep, that would be much cleaner 👍